### DISCUSSION TITLE
Adding Remark Plugins from Child Themes

### DISCUSSION THREAD

**Discussion by abrarmohi** [LekoArts/gatsby-themes#614] [Questions] [answered]
Hi there,

I just wanted to ask how can I add a new remark plugin when mdx is already declared in the core theme?

I am using gatsby-starter-minimal-blog and want to add gatsby-remark-autolink-headers. It is throwing me an error that mdx has already been declared in the parent theme.

Thanks!

**Comment by LekoArts** [comment, marked as answer]
Hi!

You'll need to use the `mdx` option set to `false` (https://github.com/LekoArts/gatsby-themes/tree/master/themes/gatsby-theme-minimal-blog#theme-options), [copy/paste the existing config](https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog-core/gatsby-config.js#L22-L48) (without the check for `options.mdx`) and then add your `gatsby-remark-autolink-headers` plugin.
